versioned/persist/tests/src/main/java/org/projectnessie/versioned/persist/tests/AbstractConcurrency.java

```diff
@@ -287,15 +293,29 @@ void concurrency(Variation variation) throws Exception {
     } finally {
       stopFlag.set(true);
 
-      System.out.printf(
-          "AbstractDatabaseAdapterTest.concurrency - %s : Commits OK: %s  Retry-Failures: %s%n",
-          variation, commitsOK, retryFailures);
-
       executor.shutdownNow();
 
       // 30 seconds is long, but necessary to let transactional databases detect deadlocks, which
       // cause Nessie-commit-retries.
       assertThat(executor.awaitTermination(30, TimeUnit.SECONDS)).isTrue();
+
+      System.out.printf(
+          "AbstractDatabaseAdapterTest.concurrency - %s : Commits OK: %s  Retry-Failures: %s%n",
```

Comment:
is `AbstractDatabaseAdapterTest` still the right name to use here?
as a newbie i dont see the immediate connection